helpCommand.js

```diff
@@ -0,0 +1,7 @@
+const helpCommand = (message) => {
```

Comment:
just like you want each file to be small and self-contained with a single purpose, you usually want each PR to be small and self-contained with a single purpose (this philosophy extends to a lot of things and originated in the Unix Philosophy https://www.slideshare.net/maryamalmarrii/unix-philosophy-and-principles https://en.wikipedia.org/wiki/Unix_philosophy upon which Mac OS and Linux are based)


-- which means this PR could be just the splitting into files, with a separate PR for the addition of the help command. In practice though, we do tend to include small unrelated stuff in PRs when we think no one is looking 😏

Reply:
the reason is so that it's easier to review, and easier to approve one piece while needing changes for the other piece